client/test/TestPorting.res

```diff
@@ -1,6 +1,8 @@
 open Prelude
 open Tester
 
+// hmmm: should this be renamed or something? it only tests String.dropRight
```

Comment:
Probably removed. This tests the things that came up when porting the client from Elm